frontend/src/fixtures/ucsbArticlesFixtures.js

```diff
@@ -0,0 +1,38 @@
+const ucsbArticlesFixtures = {
+  oneArticle: {
+    id: 1,
+    title: "Vegetables Are Better Than Fruits",
+    url: "onion.com",
+    explanation: "Nutritional values about different foods",
+    email: "raymondchen@ucsb.edu",
```

Comment:
Using your actual email for testing data is generally a bad practice. Consider something like raymond@test.com.